applications/matter_bridge/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -5,7 +5,7 @@
 #
 
 # Set 10 BLE connections, as it is an upper limit supported caused by RAM usage due to using Thread and BLE at same time.
-CONFIG_BT_MAX_CONN=10
+CONFIG_BT_MAX_CONN=9
```

Comment:
@kkasperczyk-no @LuDuda I've decreased the number of connections due to this PR:https://github.com/zephyrproject-rtos/zephyr/pull/79557. @piotrkoziar please explain why it is needed to set that config to 2k.

Reply:
Please note that it's not enough to change this Kconfig, as there are several Kconfigs below that depend on this value.